scan_test.py

```diff
@@ -383,6 +387,8 @@ def test_sns_scan_results(self):
             "MessageAttributes": {
                 "av-status": {"DataType": "String", "StringValue": scan_result},
                 "av-signature": {"DataType": "String", "StringValue": scan_signature},
+                "bucket": {"DataType": "String", "StringValue": self.s3_bucket_name},
+                "key": {"DataType": "String", "StringValue": self.s3_key_name},
```

Comment:
Same as above.

Reply:
replied in https://github.com/jobready/bucket-antivirus-function/pull/2/files#r386697550